llvm/lib/Target/AArch64/AArch64Features.td

```diff
@@ -522,6 +522,39 @@ def FeatureTLBIW : ExtensionWithMArch<"tlbiw", "TLBIW", "FEAT_TLBIW",
 //  Armv9.6 Architecture Extensions
 //===----------------------------------------------------------------------===//
 
+def FeatureCMPBR : ExtensionWithMArch<"cmpbr", "CMPBR", "FEAT_CMPBR",
```

Comment:
Hey Marian, 
If the requirements are not going to block us with the implementation of the instructions. 
I was thinking if maybe it is not better to remove all the requirements from this patch and we address this when all the dpISA is already upstream.